interfaces.go

```diff
@@ -161,10 +161,12 @@ type Exception struct {
 // An EventID must be 32 characters long, lowercase and not have any dashes.
 type EventID string
 
+type Context map[string]interface{}
```

Comment:
I'm not sure if we should introduce a concrete `Context` type. It'll force people to pass `sentry.Context` explicitly. Where if we'd use type alias, `type Context = map[string]interface{}`, it'd still work with most cases where people already pass a valid dictionary.

Reply:
I defined a new type since you mention on Slack it was a v0 and we should strive to make the API as it should. The type alias works too and the difference is minimal so that'll works well too.